solr/core/src/java/org/apache/solr/schema/IndexSchema.java

```diff
@@ -1556,48 +1547,46 @@ SimpleOrderedMap getProperties(SchemaField sf) {
         }
       }
     }
-    if (null != dynamicCopyFields) {
```

Comment:
Is this big diff just a re-indent after removing the wrapping null check?

Reply:
> Is this big diff just a re-indent after removing the wrapping null check?

Yes just removed the null check.